server.js

```diff
@@ -39,36 +39,37 @@ io.sockets.on('connection', function (socket) {
       //setTimeout(function(){ ;
       var candlestickData = financialData.candlestickData;
       var indicators = financialData.indicators;
-      callback(null, candlestickData, indicators);
+      callback(null, candlestickData, indicators, financialData);
```

Comment:
Perhaps in the future we could consolidate all these parameters into just one `financialData`.